bin/torii/src/main.rs

```diff
@@ -172,16 +175,79 @@ async fn main() -> anyhow::Result<()> {
     let world = WorldContractReader::new(args.world_address, &provider);
 
     let db = Sql::new(pool.clone(), args.world_address).await?;
+
+    let event_processors: AHashMap<Felt, Box<dyn EventProcessor<_>>> =
```

Comment:
Could we clean up this, clutters quite a lot the specifric file

Reply:
i tried to simplify this but it only compiled when i specified it fully, i was thinking using a macro to simplify this. will do it later for now i think its okay

Reply:
Putting the default hash map in a file can make more readable. 👍